sdk/Cargo.toml

```diff
@@ -93,6 +93,18 @@ tls = [ "reqwest?/rustls-tls", "rumqttc?/use-rustls" ]
 client = [ "pow", "tokio", "zeroize", "url", "reqwest", "async-trait", "log", "thiserror", "futures", "serde", "instant", "iota-crypto/bip39", "iota-crypto/bip39-en", "iota-crypto/slip10" ]
 wallet = [ "client" ]
 
+# Simple Transaction Examples
+
+[[example]]
+name = "request_funds"
+path = "examples/how-tos/simple_transaction/request_funds.rs"
```

Comment:
Can you make it `how_tos` please? Dash are not standard in folder names in rust

Reply:
Yeah just did 😆 

But currently still struggling with nodejs and formatting 😓